index.html

```diff
@@ -20,16 +42,164 @@
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+        </section>
+      </section>
```

Comment:
- [ ] I think you forgot to add the 'About  myself' section here.

Reply:
In this template, there is no 'About me' section, there are only the social media links sections repeated twice, and I thought the first instance of them would serve as the 'About me' section, tell me what you think!

Reply:
Please, can I have the link to your project requirements again? I want to show you something about it. I have been searching for it but due to the fact that I have so many reviews performed, I can't really say which one is yours.

Reply:
## Links to the project requirements and the portfolio Figma template:

- Here's the link to the requirements of the project: [Portfolio projects requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).
- Here's the link to the Figma template: [Figma template link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).

Reply:
>  I think you forgot to add the 'About myself' section here.

_In this template, there is no 'About me' section, there are only the social media links sections repeated twice, and I thought the first instance of them would serve as the 'About me' section, tell me what you think!_

**I believe the 'About me' section is merged directly with the headline section.**

Reply:
![image](https://user-images.githubusercontent.com/45668340/148533064-9f178f74-e051-4001-96d0-5264f63ac831.png)
This image means you needed to add the 'About Me' section. 
**pulled from the link above**

Reply:
I chose Template 4, the one with the dark blue background and a backdrop filter

Reply:
Should I add this section, even though it is not on my Figma design?

Reply:
I just checked. Probably, you really do not need to add the section. I just checked a friend's portfolio with your design and he has no 'About Myself' section.
Go ahead and resubmit a pull request and I believe the subsequent reviewer will approve your project if you have affected other changes.
Good luck.